alerting/src/test/kotlin/org/opensearch/alerting/settings/AlertingSettingsIT.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.settings
+
+import org.junit.Before
+import org.mockito.Mockito
+import org.opensearch.alerting.AlertingRestTestCase
+import org.opensearch.client.Client
+import org.opensearch.test.OpenSearchIntegTestCase
+
+class AlertingSettingsIT : AlertingRestTestCase() {
+    private lateinit var alertingSettings: AlertingSettings
+
+    @Before
+    fun setup() {
+        alertingSettings = AlertingSettings(OpenSearchIntegTestCase.client())
+    }
+
+    suspend fun `test acquisition of triggers client not initialized`() {
```

Comment:
Integration tests cant be using suspend functions. Also mocking shouldn't be used in integration tests.